qemu/tests/timedrift_adjust_time.py

```diff
@@ -72,7 +72,8 @@ def close_sessions(self):
         """
         logging.info("Close useless session after test")
         open_sessions = [s for s in self.open_sessions if s]
-        return map(lambda x: x.close(), open_sessions)
```

Comment:
Is this return value being used anywhere?

Reply:
I've checked that the ret did not be assigned to any vars, and as `session.close()` always returns `None`, it makes no sense to judge whether the session has been closed correctly by its ret.